scripts/ec2cleanup.py

```diff
@@ -218,8 +220,33 @@ def delete_queues(provider_mgmt, excluded_queues, output):
         logger.exception('Exception in %r', delete_queues.__name__)
 
 
+def delete_s3_bucket(provider_mgmt, bucket_name, output):
+    bucket_list = []
+    buckets = []
+    provider_name = provider_mgmt.kwargs['name']
+    try:
+        for bucket in provider_mgmt.list_s3_bucket():
+            if bucket_name in bucket:
+                buckets.append(bucket)
+        deleted_buckets = provider_mgmt.delete_s3_buckets(bucket_names=buckets)
+        if deleted_buckets:
+            for bucket in deleted_buckets:
+                bucket_list.append([provider_name, bucket])
+                logger.info("Deleted S3 Buckets: %r", bucket_list)
```

Comment:
Required:
First, It should not be in the loop. If 3 bucket deleted by `L231` then It will log three-time...

Reply:
This was due to Indentation.

Reply:
Fixed it.